setup/git.md

```diff
@@ -0,0 +1,23 @@
+# Git and Github
+
+## Git
+
+Git is a version control software, which allows you to save your code in steps, like different drafts for an essay. It also allows for branches, which could represent someone else's edits to your paper, which then you merge into your main paper.
+
+For this tutorial, git and Github are not nessicary, but highly recommended. You can download git from: https://git-scm.com/.
+
+## Github
+
+Github is an online service that allows people to store their code online for free. Github uses the git program for this. You can create an account, and once that is complete, navigate to https://github.com/fabricMC/fabric-example-mod. This example mod created by fabric is template for your mods, with everything setup.
+
+## Getting the Example mod
+
+### Without Git
+
+If you did not create a Github account or installed git, click the `Code` dropdown, then click `Download ZIP`. Extract this zip file, and then you can move onto the next tutorial.
+
+## With Git
+
+If you created a Github account, click the `Use this template` button. This will copy all the files into a new git repository for you. Give it a descriptive name for your mod. Once that is complete, open the folder you want to work on your mod from. This is where we will clone the git repository into. Once you have the folder open, on Windows right click, and you should see an option called `Git Bash Here`, click on that.
+
+After that, you will click the `Code` button in your repository and copy the url for your repository. In Git Bash, type `git clone ` then press `shift + insert`, which will paste the url into the command. After this, hit enter and wait for the repository to download. You can now move on to the next tutorial.
```

Comment:
There's a much easier option. Most IDEs have an 'import project from version control' option, which removes the need for console git commands. I know IDEA is the primary IDE targetted, so it may be worth including that process, as it's just 2 clicks and a link copy-paste.